src/App.tsx

```diff
@@ -39,6 +43,12 @@ export default function App() {
   };
 
   const handlePersonChanges = (updatedPersonData: PersonData, i: number) => {
+    for (const otherPerson of members) {
+      if (updatedPersonData.covidEvents.InHouseExposure[otherPerson.name]) {
+        otherPerson.covidEvents.InHouseExposure[updatedPersonData.name] =
```

Comment:
This does nothing as you're attempting to mutate members. You have to build the new members and then set it with setMembers. It is similar to what we did for RecordSponge with `replace` calls.

Reply:
In other words, treat the variable `members` as read-only.

Reply:
That said the calculator already accounts for asymmetries. I'm not sure if having "normalized" data, in the sense that with symmetry there is only one possible representation of the information (vs with no symmetry there could be 2 different representations), is necessary.